components/secure_embed/browser/secure_embed_host.cc

```diff
@@ -36,10 +27,10 @@ SecureEmbedHost::SecureEmbedHost(content::RenderFrameHost*) : secure_embed_() {
 }
 
 SecureEmbedHost::~SecureEmbedHost() {
-  if (guest_contents_) {
-    guest_contents_->RemoveUserData(SecureEmbedHostUserData::kKey);
-  }
   --instance_count_for_testing_;
+  if (content::SecureEmbedConnector* connector = GetConnector(); connector) {
```

Comment:
nit: this can be simplified as 
```suggestion
  if (content::SecureEmbedConnector* connector = GetConnector()) {
```

Same for a few other places in this file.